gen_host_status.yml

```diff
@@ -0,0 +1,43 @@
+- hosts: all
+  gather_facts: false
+  tasks:
+    - name: A debug msg all hosts will show except for skipped ones
+      debug:
+        msg: "Playing {{ ansible_host }}"
+      when: "'_skipped' not in ansible_host"
+
+    - name: Hosts haven't really changed, but we will say they have
+      debug:
+        msg: "I am a changed host."
+      changed_when: "'_changed' in ansible_host"
```

Comment:
> changed_when: "'_changed' in ansible_host"

The task looks like it will totally work as is, but because `when` clause filters out the hosts that shouldn't run the task, it seems like this could just be `true`

Reply:
Updated. Thanks.